browser/ui/views/speedreader/speedreader_bubble_global.h

```diff
@@ -0,0 +1,75 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_UI_VIEWS_SPEEDREADER_SPEEDREADER_BUBBLE_GLOBAL_H_
+#define BRAVE_BROWSER_UI_VIEWS_SPEEDREADER_SPEEDREADER_BUBBLE_GLOBAL_H_
+
+#include <memory>
+
+#include "brave/browser/ui/speedreader/speedreader_bubble_view.h"
+#include "chrome/browser/ui/views/location_bar/location_bar_bubble_delegate_view.h"
+#include "ui/views/metadata/metadata_header_macros.h"
+
+namespace content {
+class WebContents;
+}  // namespace content
+
+namespace ui {
+class Event;
+}  // namespace ui
+
+namespace views {
+class View;
+class Label;
+class ToggleButton;
+class StyledLabel;
+}  // namespace views
+
+namespace speedreader {
+
+class SpeedreaderTabHelper;
+
+// SpeedreaderBubbleGlobal is the bubble shown when Speedreader is enabled for
+// all pages
+class SpeedreaderBubbleGlobal : public SpeedreaderBubbleView,
```

Comment:
Maybe we could think about a better name. The bubble is not actually global, right? Maybe SpeedreaderPerSiteBubble?

Reply:
Since it is namespaced, maybe one should be called SpeedreaderBubble and the other ReaderModeBubble